lib/course_planner/notifications.ex

```diff
@@ -16,4 +17,18 @@ defmodule CoursePlanner.Notifications do
   def to(%Notification{} = notification, %User{} = user),
     do: %{notification | user: user}
 
+  def send_all_notifications do
+    Timex.today()
```

Comment:
Do we need to have one system setting like ENABLE_NOTIFICAITON that completely stop all notifications ?